tests/json_test/README.md

```diff
@@ -9,7 +9,7 @@ Test CITA using [Tests](https://github.com/cryptape/cita-testdata/)
 ```sh
 $ cd cita
 
-$ cargo test state_test::tests::test_json_state
+$ cargo test --features sha3hash state_test::tests::test_json_state
```

Comment:
No need to use `--features sha3hash`. because `Cargo.toml` already has the default features.
That's OK